zebra-instructions.md

```diff
@@ -26,7 +26,7 @@ Install dependencies:
 Install `s-nomp`:
 1. `git clone https://github.com/ZcashFoundation/s-nomp`
 2. `cd s-nomp`
-3. Use the Zebra configs: `git checkout zebra-mining`
+3. Use the Zebra configs: `git checkout -t origin/zebra-mining`
```

Comment:
I have changed the default branch on this repository, so new clones should automatically clone and checkout the zebra-mining branch.